README.md

```diff
@@ -1,14 +1,14 @@
-# AI Pipelines
-AI Pipelines is a tool for generating end-to-end pipelines composed of GCP components so that any customer can easily migrate their local ML models onto GCP and start realizing the benefits of the cloud quickly. Currently ML pipelines are very difficult to implement for customers, and take weeks if not months with experienced Googlers.
+# ML Pipeline Generator
+ML Pipeline Generator is a tool for generating end-to-end pipelines composed of GCP components so that users can easily migrate their local ML models onto GCP and start realizing the benefits of the Cloud quickly. 
 
 The following ML frameworks will be supported:
-1. Tensorflow (TF)
+1. TensorFlow (TF)
 1. Scikit-learn (SKL)
 1. XGBoost (XGB)
 
-We will first only consider Kubeflow Pipelines (KFP) for orchestrating ML pipelines built using various Cloud AI Platform (CAIP) features. Orchestration using Cloud Composer (CC) may be in scope in the future.
-
-The full project plan can be found [here](https://docs.google.com/document/d/11-ljj4D3UT-_bOyFeN_L_uRXQkM9G10bte9jy1yfSYA/edit?ts=5df59215).
+The following backends are currently supported for model training:
+1. [Google Cloud AI Platform](https://cloud.google.com/ai-platform) 
+1. [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs) (managed Kubeflow Pipelines)
 
 ## Setup
```

Comment:
optional: It would be cool if we also told the user which APIs to enable and provide links.

Reply:
ack, will add in a new commit later.